erpnext_com/www/docs/user/manual/en/setting-up/automation/auto-repeat.md

```diff
@@ -0,0 +1,71 @@
+<!-- add-breadcrumbs -->
+# Auto Repeat
+
+Auto Repeat feature helps you create certain documents automatically in a given time period.
+
+You can now Customize any Form to make the documents **repeatable**.
+
+For Example: Assuming that you follow deferred expense system for some items. It requires you to create same Journal Entry every month to credit Deferred Expense account and debit Expense Account. You can create first Journal Entry manually for it, and then create auto-repeat transaction for it.
+
+## 1. How to set up Auto Repeat
+
+### 1.1 Customize the Form
+1. Go to: **Home > Customization > Form Customization > Customize Form**.
+2. Select the form in which you want to allow creation of repeatable documents.
+3. Check 'Allow Auto Repeat' to allow the creation of repeatable documents for that Form.
+
+<img class="screenshot" alt="Allow Auto Repeat" src="/docs/assets/img/setup/automation/allow-auto-repeat.png">
+
+### 1.2 Set up Auto Repeat
+1. Go to **Home > Settings > Auto Repeat > New**.
```

Comment:
```suggestion
1. Go to the Auto Repeat list, click on new.
```

Reply:
Added a new section 'Automation' to the Settings Dashboard. Made changes in docs accordingly.